Pixel Game/Assets/Scripts/NpcData.cs

```diff
@@ -16,11 +16,15 @@ public class NpcData : MonoBehaviour
     private GameObject dialogueCanvas;
     private GameObject mainCamera;
 
+    private void Awake()

```

Comment:
Any reason to do this in awake instead of start? :)

Reply:
Yeah, in some cases `dialogueCanvas` would be null if not assigned in Awake. Not sure why though.

Reply:
Got it!